app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for i in people:
+        Person(i["name"], i["age"])
+
+    for i in people:
+        name = i["name"]
+        human = Person.people[name]
+
+        if "wife" in i and i["wife"]:
+            human.wife = Person.people[i["wife"]]
+            if hasattr(human, "husband"):
+                del human.husband
```

Comment:
The check and deletion of the `husband` attribute when setting a `wife` is redundant. Since the `husband` attribute is not set when a `wife` is being assigned, this block can be removed to simplify the code.